test/integration/deprecations-test.js

```diff
@@ -3,8 +3,10 @@ const { mock } = require("simple-mock");
 
 test("@octokit/webhooks", (t) => {
   const emitWarningMock = mock(process, "emitWarning");
-  const createWebhooksApi = require("../../src");
```

Comment:
Please revert these changes

Reply:
It wasn't meant for the new exporting format. Just the style/formatting of the file itself

Reply:
> It wasn't meant for the new exporting format. Just the style/formatting of the file itself

Ah ok. But still not clear what should I restore. Running `lint` I don't get any error or warning.

Reply:
It fixes any issues automatically. It's fixed in your later commits